File: Patterns/2. Rectangle Patterns/12. Hollow Rectangle Pattern/hrp.py
l = int(input("Enter the Length : "))
b = int(input("Enter the Breadth : "))

for i in range(1,l+1):
    for j in range(1,b+1):
        if(i==1 or i==l or j==1 or j==b):
            print("*", end=' ')
        else:
            print(end='  ')
    print()

'''
Output:

* * * * * * * 
*           * 
*           * 
*           * 
* * * * * * * 
'''

File: Patterns/02. Rectangle Patterns/12. Hollow Rectangle Pattern/hrp.cpp
#include <iostream>
using namespace std;
int main(){
    int l,b;
    cout<<"Enter the Length : ";
    cin>>l;
    cout<<"Enter the Breadth : ";
    cin>>b;

    for(int i=1; i<=l; i++){
        for(int j=1; j<=b; j++){
            if(i==1 or i==l or j==1 or j==b){
                cout<<"* ";
            }else{
                cout<<"  ";
            }
        }
        cout<<'\n';
    }
}

/*
Output

* * * * * * * 
*           * 
*           * 
*           * 
* * * * * * *

*/